packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -380,18 +402,13 @@ static void interpretGoogleMapOptions(Object o, GoogleMapOptionsSink sink) {
     if (buildingsEnabled != null) {
       sink.setBuildingsEnabled(toBoolean(buildingsEnabled));
     }
-    final Object enableMarkerCaching = data.get("enableMarkerCaching");
-    if (enableMarkerCaching != null) {
-      sink.setMarkerCachingEnabled(toBoolean(enableMarkerCaching));
-    }
     final Object markersAnimationEnabled = data.get("markersAnimationEnabled");
     if (markersAnimationEnabled != null) {
       sink.setMarkersAnimationEnabled(toBoolean(markersAnimationEnabled));
     }
   }
 
-  /** Returns the dartMarkerId of the interpreted marker. */
-  static String interpretMarkerOptions(Object o, MarkerOptionsSink sink, CozyMarkerBuilder cozyMarkerBuilder) {
+  static String interpretMarkerOptionsWithoutIcon(Object o, MarkerOptionsSink sink) {
     final Map<?, ?> data = toMap(o);
```

Comment:
Seems it's worth to define the map type here also (same a Rafa's comment)

Reply:
To change this, I would need to change the function as well (otherwise it would be just a cast), and it would change in a lot other places. Seeing the commit log of plugins repo, it has been like that for a while (maybe for some reason), so I will not bother to change this and possibly break something